Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -24,7 +24,7 @@ Other tools are really a matter of preference and budget. A ton of free and comm
 
 #### Setting up the Android SDK
 
-Local Android SDK installations are managed through Android Studio. Create an empty project in Android Studio and select "Tools->Android->SDK Manager" to open the SDK Manager GUI. The "SDK Platforms" tab lets you install SDKs for multiple API levels. Recent API levels are:
+Local Android SDK installations are managed through Android Studio. Select "Configure->SDK Manager" from the initial Android Studio window, or create an empty project in Android Studio and select "Tools->Android->SDK Manager" to open the SDK Manager GUI. The "SDK Platforms" tab lets you install SDKs for multiple API levels. Recent API levels are:
```

Comment:
This is a long sentence with the actual goal all the way at the end. I think it would be better to put it at the front: "In order to open the SDK Manager GUI, select ...".

Reply:
I agree. I just followed (or complemented) the already existing sentence, but definitely you can reword it.